src/validator/data_model.py

```
@@ -49,43 +49,249 @@ def _clean_key(self, key: str) -> str:
+                validated_props = StandardMaterialProperties.model_validate(properties)
+                data['Properties'] = validated_props
+            elif material_type == "NoMass":
+                validated_props = NoMassMaterialProperties.model_validate(properties)
+                data['Properties'] = validated_props
+            elif material_type == "AirGap":
+                validated_props = AirGapMaterialProperties.model_validate(properties)
+                data['Properties'] = validated_props
+            elif material_type == "Glazing":
+                validated_props = GlazingMaterialProperties.model_validate(properties)
+                data['Properties'] = validated_props
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)